camptix.js

```diff
@@ -179,12 +179,21 @@ var docCookies={getItem:function(e){return decodeURIComponent(document.cookie.re
 		},
 
 		init: function() {
-			if ('undefined' == typeof $.fn.appear || lazyLoad.cache.$attendees.length < 1) {
+			// Dependencies
+			if (
+				lazyLoad.cache.$attendees.length < 1
+				||
+				'undefined' == typeof $.fn.appear
+				||
+				'undefined' == typeof wp
+			) {
```

Comment:
This is a bit different than the coding standards:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#multi-line-statements

I don't do a good job of following that one, either, though. In general, I think up to 115 characters per line is ok these days. but there are lots of different opinions. This would be 123, though, so it would be good to break it up.

I think there a couple strategies that might work better than multiple lines inside the `if()`. When a condition has non-trivial logic, it can be nice to just assign that to a variable w/ a descriptive name, so the statement can be more like `if ( someConditionMet && ! someOtherCondition && aThirdCondition )`, but in this case the logic is pretty simple, so that doesn't feel great.

In this case, my favorite way would probably  be better to break it into two `if( ... ) { return }` blocks; one for the attendees length, and a second for the `typeof` checks.

I don't think we need to be very strict about this particular rule, or, if we do, then I need to start changing my habits too :) So, if you like any of the alternatives better, that's cool, but if not it's totally fine to leave it like this.